### PR TITLE
#2rbzdym - Using mixins

### PR DESCRIPTION
**Pull request by IzzXr**
## Task

Issue: #2rbzdym

## Decision

> Please describe task description in more detail.
> If it was a bug, then describe what was causing the problem.

refactoring  styles

## Screenshots

> If you changed the UI, then you need to attach a screenshot/video.

### CODE REVIEW

**Comment by alexeyshirko** [comment]
Task linked: [CU-2rbzdym Using mixins](https://app.clickup.com/t/2rbzdym)

File: src/core/components/ui/Icon/style/Icon.module.scss [hunk 1]
```diff
@@ -1,5 +1,6 @@
+@import "./../../../../../assets/style/mixins";
```
Comment (alexeyshirko) [inline src/core/components/ui/Icon/style/Icon.module.scss:1]: use `use` instead of `import` with an absolute path, it is shorter and always the same

**Review by alexeyshirko — CHANGES_REQUESTED** [review]

File: src/core/components/ui/Button/style/Button.module.scss [hunk 2]
```diff
@@ -1,11 +1,12 @@
+@use "src/assets/style/mixins" as mixin  ;
```
Comment (alexeyshirko) [inline src/core/components/ui/Button/style/Button.module.scss:1]: `as mixin  ;` what is it?

Comment (IzzXr) [reply src/core/components/ui/Button/style/Button.module.scss:1]: use name mixins call an error, so I assigned a different name for the mixin, or is it called namespace control in another way

File: src/views/dispatcher/modules/Topbar/style/Topbar.module.scss [hunk 3]
```diff
@@ -1,13 +1,12 @@
+@import "./../../../../../assets/style/mixins";
```
Comment (alexeyshirko) [inline src/views/dispatcher/modules/Topbar/style/Topbar.module.scss:1]: `use`, not `import`

File: src/core/components/ui/Button/style/Button.module.scss (hunk 2 above)
Comment (alexeyshirko) [inline src/core/components/ui/Button/style/Button.module.scss:1]: remove space

**Review by alexeyshirko — CHANGES_REQUESTED** [review]

**Review by alexeyshirko — APPROVED** [review]

**Pull request merged**